pyrate/core/dem_error.py

```diff
@@ -43,18 +43,22 @@ def remove_dem_error(ifgs: List, params: dict) -> None:
     ifg0 = Ifg(ifg0_path)
     ifg0.open(readonly=True)
 
-    # calculate per-pixel lon/lat
-    lon, lat = get_lonlat_coords(ifg0)
-    # calculate per-pixel radar coordinates
-    az, rg = get_radar_coords(ifg0, params)
-    # calculate per-pixel look angle (also calculates and saves indcidence and azimuth angles)
-    look_angle = calc_local_geometry(ifg0, ifg0_path, az, rg, lon, lat, params)
-    # loop over all Ifgs in stack
-    ifgs = [shared.Ifg(p) for p in ifg_paths] if isinstance(ifgs[0], str) else ifgs
-    process_ifgs = mpiops.array_split(ifgs)
-    for ifg in process_ifgs:
-        bperp = calc_local_baseline(ifg, az, look_angle, params)
-    # todo: add code to invert for DEM error using the per-pixel baseline values and the unwrapped phase
+    # not currently implemented for ROIPAC data which breaks some tests
+    # if statement can be deleted once ROIPAC is deprecated from PyRate
+    if not ifg0.meta_data[ifc.PYRATE_INSAR_PROCESSOR] == 'ROIPAC':
```

Comment:
Just a comment. By the time the code progresses to this module, it should be input-format agnostic. Meaning that preprocessing steps have grabbed what is needed from Gamma, ISCE, SNAP formats and converted it to a standard "Pyrate" format that this module expects.

Reply:
Agree. I only needed to add this line due to some ROIPAC tests failing.